webanno-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/curation/storage/CurationDocumentService.java

```diff
@@ -67,19 +67,6 @@ CAS readCurationCas(SourceDocument document)
 
     void deleteCurationCas(SourceDocument document)
             throws IOException;
-
-    /**
-     * Remove a curation annotation document from the file system, for this {@link SourceDocument}
-     *
-     * @param sourceDocument
-     *            the source document.
-     * @param username
-     *            the username.
-     * @throws IOException
-     *             if an I/O error occurs.
-     */
-    void removeCurationDocumentContent(SourceDocument sourceDocument, String username)
```

Comment:
From my understanding, `removeCurationDocumentContent` was used here exactly because it does not remove the CAS from the session but only clears the underlying file. The idea was (probably) that we keep a lock on the CAS in the session to avoid a concurrent thread from obtaining a lock and doing something with the CAS (e.g. regenerating it) while we are still in the re-merge process.